wagtail/admin/views/pages/bulk_actions/publish.py

```diff
@@ -32,19 +32,15 @@ def get_execution_context(self):
         }
 
     @classmethod
-    def execute_action(cls, objects, **kwargs):
-        include_descendants = kwargs.get('include_descendants', False)
-        user = kwargs.get('user', None)
-        if user is None:
-            return
+    def execute_action(cls, objects, include_descendants=False, user=None):
```

Comment:
Should we change the base class `get_execution_context()` output, this will explode with a `TypeError`. 

suggestion: leave `**kwargs` in since `BulkAction` has the following method definition: `def execute_action(cls, objects, **kwargs)`